yb-voyager/cmd/common.go

```diff
@@ -201,22 +201,43 @@ func displayExportedRowCountSnapshotAndChanges() {
 	if err != nil {
 		utils.ErrExit("failed to read export status during data export snapshot-and-changes report display: %v", err)
 	}
+	sourceSchemaCount := len(strings.Split(source.Schema, "|"))
 	for i, tableStatus := range exportStatus.Tables {
+
+		schemaName := tableStatus.SchemaName
+		if sourceSchemaCount <= 1 {
+			schemaName = ""
+		}
 		if i == 0 {
-			uitable.AddRow(headerfmt("SCHEMA"), headerfmt("TABLE"), headerfmt("SNAPSHOT ROW COUNT"), headerfmt("TOTAL CHANGES EVENTS"),
-				headerfmt("INSERTS"), headerfmt("UPDATES"), headerfmt("DELETES"),
-				headerfmt("FINAL ROW COUNT(SNAPSHOT + CHANGES)"))
+			if schemaName == "" {
+				uitable.AddRow(headerfmt("TABLE"), headerfmt("SNAPSHOT ROW COUNT"), headerfmt("TOTAL CHANGES EVENTS"),
+					headerfmt("INSERTS"), headerfmt("UPDATES"), headerfmt("DELETES"),
+					headerfmt("FINAL ROW COUNT(SNAPSHOT + CHANGES)"))
+			} else {
+				uitable.AddRow(headerfmt("SCHEMA"), headerfmt("TABLE"), headerfmt("SNAPSHOT ROW COUNT"), headerfmt("TOTAL CHANGES EVENTS"),
```

Comment:
What if we don't have a separate column for `SCHEMA` and repurpose `TABLE` column to mean "qualified table name"?
Then if `schemaName` is empty-string, we don't need to qualify the table name. This will allow us to get rid of all the special casing happening to display two separate tables (one with schema and one without).